core/templates/domain/summary/exploration-summary-backend-api.service.ts

```diff
@@ -62,9 +62,8 @@ export class ExplorationSummaryBackendApiService {
       explorationIds: string[],
       includePrivateExplorations: boolean,
       successCallback: (
-        value?: ExplorationSummaryBackendDict |
-        Promise<ExplorationSummaryBackendDict>) => void,
-      errorCallback: (reason?: string | string[]) => void): void {
+        value: ExplorationSummaryBackendDict) => void,
```

Comment:
you may want to make the exact same formatting changes in this file as made [here](https://github.com/oppia/oppia/pull/13520/files#diff-0ac75837eefa3bfcea0c010f125184b3fbfa27e1e0bc591b6aad9d0f1563f8bd)

Reply:
done